Controllers/address.js

```diff
@@ -0,0 +1,50 @@
+const _ = require('lodash');
+exports.getAddress = async (req, res) => {
+	const { address } = req;
+	try {
+		res.json({ response: address });
+	} catch {
+		res.status(401).json({ response: error.message });
+	}
+};
+
+exports.addAddress = async (req, res) => {
+	const { address } = req;
+	const addAddress = req.body;
+	try {
+		const updatedAddress = _.extend(address, {
```

Comment:
I have a Suggestion: `_.extend` kind of confusing
instead you can do:
```
const { extend } = require('lodash');
extend(address, addressUpdates)
```

Reply:
done